UIKitLesson1/Helper/PaymentVC+.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  PaymentVC+.swift
+//  UIKitLesson1
+//
+//  Created by Kirill Dev on 19.01.2023.
+//
+
+import Foundation
+
+import UIKit
+
+extension PaymentViewController {
```

Comment:
```suggestion
extension UIViewController {
```
уже помоему была такая ошибка